core-bundle/src/Resources/contao/widgets/KeyValueWizard.php

```diff
@@ -66,7 +66,7 @@ public function validate()
 			foreach ($options as $key=>$option)
 			{
 				// Unset empty rows
-				if (!$option['key'])
+				if (trim($option['key']) === '')
```

Comment:
Is `$option['key']` always set? Otherwise we should probably add `!isset($option['key']) || `.

Reply:
I think it's fine this way. If no values are passed, an array with empty values is returned:
![image](https://user-images.githubusercontent.com/48379929/118452874-db286600-b6f6-11eb-9ac8-8990d0d92ec7.png)

Reply:
```suggestion
				if (trim($option['key'] ?? '') === '')
```

Reply:
From my point of view `$option['key']` can never be `null` in the `validate` method. Should I still take it that way?

Reply:
Agree, `trim($option['key']) === ''` should be fine then.